.github/workflows/image.yml

```diff
@@ -0,0 +1,45 @@
+name: image
+
+on:
+  pull_request:
+    branches:
+      - 'main'
+      - 'release-*'
+  push:
+    tags:
+      - 'v*.*.*'  
```

Comment:
Is semver common in actions? Most actions I've seen use integer versioning, or is there some other magic happening there?

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushbranchestagsbranches-ignoretags-ignore

Reply:
This doesn't answer my question. I was asking whether we're using semver for the action even though other actions use integer versioning.

I suppose our versioning is tied to the holodeck version.

Reply:
I want to use SemVer so is easier to use this action , but is a personal preference, if the team doesn't feel that way, I can edit, not a blocker